contentcuration/contentcuration/frontend/channelEdit/pages/StagingTreeView/index.vue

```diff
@@ -1,229 +1,248 @@
 <template>
 
-  <VContainer fluid fill-height class="pa-0">
-    <LoadingText v-if="isLoading" />
-    <VLayout
-      v-else-if="isEmpty"
-      justify-center
-      fill-height
-      style="padding-top: 10%;"
-    >
-      <VFlex class="text-xs-center">
-        <h1 class="headline font-weight-bold mb-2">
-          {{ $tr('emptyChannelText') }}
-        </h1>
-        <p class="subheading">
-          {{ $tr('emptyChannelSubText') }}
-        </p>
-      </VFlex>
-    </VLayout>
-
-    <VLayout v-else row>
-      <ResizableNavigationDrawer
-        permanent
-        clipped
-        localName="topic-staging-tree"
-        class="hidden-xs-only"
-        :maxWidth="400"
-        :minWidth="200"
-        :style="{ backgroundColor: $vuetify.theme.backgroundColor }"
-      >
-        <VLayout row>
-          <IconButton
-            icon="collapse_all"
-            :text="$tr('collapseAllButton')"
-            @click="collapseAll"
-          >
-            $vuetify.icons.collapse_all
-          </IconButton>
-          <VSpacer />
-          <IconButton
-            :disabled="!ancestors || !ancestors.length"
-            icon="gps_fixed"
-            :text="$tr('openCurrentLocationButton')"
-            @click="jumpToLocation"
-          />
+  <div>
+    <ToolBar v-if="currentChannel" color="white">
+      <VToolbarSideIcon @click="drawer = true" />
+      <VToolbarTitle>
+        {{ $tr('deploy') }} <span class="notranslate">{{ currentChannel.name }}</span>
+        <router-link :to="rootTreeRoute" class="pl-2 body-1" data-test="root-tree-link">
+          {{ $tr('backToViewing') }}
+        </router-link>
+      </VToolbarTitle>
+
+      <VSpacer />
+      <OfflineText indicator />
+      <span class="grey--text grey--darken-2">{{ $tr('reviewMode') }}</span>
+    </ToolBar>
+    <MainNavigationDrawer v-model="drawer" />
+
+    <VContent>
+      <VContainer fluid fill-height class="pa-0">
```

Comment:
I need to double check on this, but I think the topic tree hierarchy drawer, resource drawer, and current topic list should all have their own scrollbars instead of one for the whole page. Probably not a big deal if we follow this up in a separate PR though
![image](https://user-images.githubusercontent.com/7447496/88349580-54d50a80-cd05-11ea-8b40-b0bd7cb1371c.png)

Reply:
That makes sense. I opened [an issue]([url](#2000
)) because I guess we should update the studio tree component and the resource drawer component that will also affect other places in the app.